Can't run "npm run start"

When I run `npm run start` after `npm install` there is an error
`Starting '<anonymous>'...
[10:10:56] '<anonymous>' errored after 12 ms
[10:10:56] Error: spawn ./node_modules/.bin/electron ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:232:19)
    at onErrorNT (internal/child_process.js:407:16)
    at process._tickCallback (internal/process/next_tick.js:63:19)
[10:10:56] 'start' errored after 856 ms`

i have the same error

> [10:04:32] Finished 'build-css' after 34 ms
[10:04:32] Starting 'build-js'...
[10:04:32] Finished 'build-js' after 504 ms
[10:04:32] Finished 'build' after 545 ms
[10:04:32] Starting '<anonymous>'...
[10:04:32] '<anonymous>' errored after 6.49 ms
[10:04:32] Error: spawn ./node_modules/.bin/electron ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:190:19)
    at onErrorNT (internal/child_process.js:362:16)
    at _combinedTickCallback (internal/process/next_tick.js:139:11)
    at process._tickDomainCallback (internal/process/next_tick.js:219:9)
[10:04:32] 'start' errored after 663 ms
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! electrate@1.0.0 start: `gulp start`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the electrate@1.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:

--- a/.babelrc
+++ b/.babelrc
@@ -1,3 +1,6 @@
 {
-  "presets": ["es2015", "react"]
+  "presets": ["@babel/env", "@babel/react"],
+  "plugins": [
+    "@babel/plugin-proposal-class-properties"
+  ]
 }

--- a/gulpfile.js
+++ b/gulpfile.js
@@ -3,6 +3,7 @@ const gulp  = require('gulp');
 const maps  = require('gulp-sourcemaps');
 const babel = require('gulp-babel');
 const css   = require('gulp-css');
+const path   = require('path');
 
 /* Build */
 
@@ -40,21 +41,21 @@ gulp.task('copy', gulp.parallel('copy-html', 'copy-assets'));
 
 /* Execute */
 
-const cmd   = (name) => `./node_modules/.bin/${name}`;
+const cmd   = (name) => path.join('.', 'node_modules', '.bin', name);
 const args  = (more) => Array.isArray(more) ? ['.'].concat(more) : ['.'];
 const exit  = () => process.exit();
 
 gulp.task('start', gulp.series('copy', 'build', () => {
-    spawn(cmd('electron'), args(), { stdio: 'inherit' }).on('close', exit);
+    spawn(cmd('electron'), args(), { stdio: 'inherit', cwd: '.', shell: true }).on('close', exit);
 }));
 
 
 gulp.task('release', gulp.series('copy', 'build', () => {
-    spawn(cmd('electron-builder'), args(), { stdio: 'inherit' }).on('close', exit);
+    spawn(cmd('electron-builder'), args(), { stdio: 'inherit', cwd: '.', shell: true }).on('close', exit);
 }));
 
 gulp.task('test', gulp.series('copy', 'build', () => {
-    spawn(cmd('jest'), args(), { stdio: 'inherit' }).on('close', exit);
+    spawn(cmd('jest'), args(), { stdio: 'inherit', cwd: '.', shell: true }).on('close', exit);
 }));
 
 

--- a/main.js
+++ b/main.js
@@ -9,7 +9,13 @@ const BrowserWindow = electron.BrowserWindow;
 let mainWindow;
 
 const createWindow = () => {
-  mainWindow = new BrowserWindow({width: 800, height: 600});
+  mainWindow = new BrowserWindow({
+    width: 800, 
+    height: 600, 
+    webPreferences: {
+      nodeIntegration: true
+    }
+  });
 
   mainWindow.loadURL(url.format({
     pathname: path.join(__dirname, 'index.html'),

--- a/package.json
+++ b/package.json
@@ -21,26 +21,27 @@
   "author": "GitHub",
   "license": "CC0-1.0",
   "devDependencies": {
-    "babel-core": "^6.26.0",
-    "babel-jest": "^22.4.1",
-    "babel-preset-env": "^1.6.1",
-    "babel-preset-es2015": "^6.24.1",
-    "babel-preset-react": "^6.24.1",
-    "electron": "~1.8.2",
-    "electron-builder": "^20.0.8",
+    "@babel/core": "^7.4.4",
+    "@babel/plugin-proposal-class-properties": "^7.4.4",
+    "@babel/preset-env": "^7.4.4",
+    "@babel/preset-react": "^7.0.0",
+    "babel-jest": "^24.8.0",
+    "electron": "~5.0.1",
+    "electron-builder": "^20.39.0",
     "electron-react-devtools": "^0.5.3",
-    "gulp": "^4.0.0",
-    "gulp-babel": "^7.0.1",
+    "gulp": "^4.0.2",
+    "gulp-babel": "^8.0.0",
     "gulp-concat": "^2.6.1",
     "gulp-css": "^0.1.0",
-    "gulp-sourcemaps": "^2.6.4",
-    "jest": "^22.4.2",
-    "react-test-renderer": "^16.2.0"
+    "gulp-sourcemaps": "^2.6.5",
+    "jest": "^24.8.0",
+    "react-test-renderer": "^16.8.6"
   },
   "dependencies": {
-    "electron-is-dev": "^0.3.0",
-    "react": "^16.2.0",
-    "react-dom": "^16.2.0"
+    "electron-is-dev": "^1.1.0",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "react-router-dom": "^5.0.0"
   },
   "build": {
     "appId": "com.karmadust.electrate",

